src/tamil/MissingTam.gf

```diff
@@ -0,0 +1,561 @@
+resource MissingTam = open GrammarTam, Prelude in {
```

Comment:
You can remove this file altogether, and generate it again when you will actually need it. Instructions on how to generate a Missing module are in 
 https://inariksit.github.io/gf/2018/08/28/gf-gotchas.html#generating-missingxxx